proc_IR.py

```diff
@@ -50,6 +42,7 @@
     fl.next('frequency domain after')
     s.ft('t2')
     fl.image(s)
+    s.ift('t2')
     #}}}
     #{{{zeroth order phase correction
     ph0 = s['t2':0]['ph2',coh_sel['ph2']]['ph1',coh_sel['ph1']]
```

Comment:
so the changes above and below have to do with the fact that previously, you were making the selection `['t2':0]` in the time domain, but with your edits, you started to make it in the frequency domain.  I'm actually extremely surprised that this didn't mess up the phasing of your data, so also be more careful when running your checks